### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by mohannadhanafi**
Make the error pages and edits on app.js component

### CODE REVIEW

File: src/client/src/components/App.js [hunk 1]
```diff
@@ -124,7 +129,9 @@ class App extends Component {
                       component={ParticipantDateDetails}
                     />
                     <Route
-                      exact
+                      exactLogout
+
```
Comment (ishak52) [inline src/client/src/components/App.js]: delete white line

Comment (mohannadhanafi) [reply src/client/src/components/App.js]: Done :+1:

File: src/client/src/components/App.js [hunk 2]
```diff
@@ -124,7 +129,7 @@ class App extends Component {
                       component={ParticipantDateDetails}
                     />
                     <Route
-                      exact
+                      exactLogout
```
Comment (tomduggan85) [inline src/client/src/components/App.js]: Was this intended?

File: src/client/src/components/pages/login/index.jsx [hunk 3]
```diff
@@ -10,7 +10,9 @@ class LoginForm extends Component {
     username: "",
     password: ""
   };
-
+  componentWillMount() {
+    this.props.history.push("/login");
```
Comment (tomduggan85) [inline src/client/src/components/pages/login/index.jsx]: Why did you need this redirect?
